README.md

```diff
@@ -44,11 +44,10 @@ Login parameters:
 - code: [string] email code for two-step verification (make sure to use string if your code starts with 0).
 
 ```php
-use FUTApi\Api\Core;
-use FUTApi\Exception\FutError;
-use FUTApi\Api\Authentication\Account;
-use FUTApi\Api\Authentication\Credentials;
-use FUTApi\Api\Authentication\Session;
+use FShapecode\FUT\Client\Api\Core;
```

Comment:
isn't this a typo? change`use FShapecode\FUT\Client\Api\Core;` to `use Shapecode\FUT\Client\Api\Core;`